### ISSUE TITLE
codesign --deep is deprecated

### ISSUE THREAD

**Issue by kerner1000** [fvarrui/JavaPackager#245]
I'm submitting a…

- [X] bug report
- [ ] feature request
- [ ] other

According to this doc ([developer.apple.com](https://developer.apple.com/forums/thread/128166)), the `--deep` option is deprecated and does not work reliably. From my personal experience, the following works better:

```
#!/bin/bash
find "../MyApp.app" -depth -type f -exec codesign -v -f -o runtime -s '3rd Party Mac Developer Application: Max Mustermann (6W37Y2F4CM)' --entitlements 'assets/mac/entitlements.plist' {} \; 

codesign -f -o runtime --entitlements 'entitlements.plist' -s '3rd Party Mac Developer Application: Max Mustermann (6W37Y3F4CM)' 'MyApp.app/Contents/MacOS/launcher'

codesign -f -o runtime --entitlements '/Volumes/Case-sensitive Volume/sources/drrename/assets/mac/entitlements.plist' -s '3rd Party Mac Developer Application: Max Mustermann (6W37Y3F4CM)' '../MyApp.app' 
```

**Comment by kerner1000** [comment]
Working on this one.

**Comment by fvarrui** [comment]
Fixed in PR #252

**Comment by fvarrui** [comment]
JavaPackager 1.7.0 released to Maven Central
